src/rust/engine/fs/store/src/remote.rs

```diff
@@ -109,18 +112,6 @@ impl ByteStore {
     )
   }
 
-  fn call_option(&self) -> grpcio::CallOption {
```

Comment:
you need to rebase on top of #8529 I think

Reply:
Yeah :) If you take a look at just the top commit, that should have just this change